src/aircraft_cmd.cpp

```diff
@@ -1586,7 +1588,7 @@ static void AircraftEventHandler_AtTerminal(Aircraft *v, const AirportFTAClass *
 			return;
 		default:  // orders have been deleted (no orders), goto depot and don't bother us
 			v->current_order.Free();
-			go_to_hangar = Station::Get(v->targetairport)->airport.HasHangar();
+			go_to_hangar = true;
```

Comment:
What behavior does this change? Does it try a different airport? Maybe the comment above this could better explain what happens in this case. 🙂

Reply:
This is a no change, if you read subsequent lines of code.

I will create another PR suggesting this.

Reply:
Thanks for the other PR. That sort of cleanup is helpful.